src/utils.js

```diff
@@ -385,6 +385,22 @@ export function basicCompatVersionComparison(versionAdded, minVersion) {
   return !Number.isNaN(asNumber) && asNumber > minVersion;
 }
 
+function firstStableVersion(supportInfo) {
+  return supportInfo.reduce((versionAdded, supportEntry) => {
+    if (
+      !Object.prototype.hasOwnProperty.call(supportEntry, 'flags') &&
+      (!versionAdded ||
+        !basicCompatVersionComparison(
+          supportEntry.version_added,
+          parseInt(versionAdded, 10)
+        ))
+    ) {
+      return supportEntry.version_added;
+    }
+    return versionAdded;
+  }, false);
```

Comment:
Nit, @freaktechnik given that I see that the type of the `versionAdded` property (as used in the mdn/browser-compat-data repo) can be both a boolean or a string, and so this function may as well return a boolean or a string, I wouldn't mind if we would make it more explicitly visible by specify in an inline comment what this function is going to return ;-)